src/TopFilters/index.js

```diff
@@ -1,108 +1,126 @@
+/* eslint-disable react/jsx-curly-newline */
+/* eslint-disable react/jsx-indent */
+/* eslint-disable react/jsx-wrap-multilines */
 /* eslint-disable react/no-array-index-key */
-/* eslint-disable prettier/prettier */
 /* eslint-disable react/jsx-filename-extension */
 /* eslint-disable react/prop-types */
 /* eslint-disable react/prefer-stateless-function */
 
-import React, { Component } from 'react';
+import React from 'react';
 import clsx from 'clsx';
+import { Switch, FormControlLabel, makeStyles } from '@material-ui/core';
```

Comment:
```suggestion
import React from 'react';

import clsx from 'clsx';

import { Switch, FormControlLabel, makeStyles } from '@material-ui/core';

```

import order is documented here: https://centreon.atlassian.net/wiki/spaces/RD/pages/141230093/Frontend